src/simplay-jupyter/src/index.ts

```diff
@@ -1,5 +1,100 @@
-// Copyright (c) team-simplay
-// Distributed under the terms of the Modified BSD License.
+import { JSONObject } from '@lumino/coreutils';
+import { Widget } from '@lumino/widgets';
+import { IRenderMime } from '@jupyterlab/rendermime-interfaces';
+import { SimulationSpooler } from 'simplay-web';
 
-export * from './version';
-export * from './widget';
+import '../style/index.css';
+import { SimulationDataSerialized } from 'simplay-web/dist/SimulationDataSerialized';
+
+const SIMPLAY_CSS_COMMON_CLASS = 'jp-render-simplay';
+
+/**
+ * The MIME type for simplay.
+ *
+ */
+export const SIMPLAY_MIME_TYPE = 'application/simplay+json';
+
+/**
+ * A widget for rendering SimPlay, for usage with rendermime.
+ */
+export class RenderSimplay extends Widget implements IRenderMime.IRenderer {
+  constructor(options: IRenderMime.IRendererOptions) {
+    super();
+    console.info('JupyterLab extension simplay_jupyter is activated!');
```

Comment:
We should think about removing the console.log

Reply:
Currently thats checked with the ui-test, I think this can stay because it indicates to a jupyter user in the console.log that an extension is active.